docs/config_manual_config.md

```diff
@@ -0,0 +1,31 @@
+# Configuration - Manually Change Files
+User configuration should be set in the `XDG_CONFIG_HOME` file.  Usually located at `~/.config/mycroft/mycroft.conf`.  This file may or may not exist by default.  If it does NOT exist, create it.
+
+`mkdir -p ~/.config/mycroft`
+
+`touch ~/.config/mycroft/mycroft.conf`
+
+Now you can edit that file.  To continue with the previous example, we will change the host of the TTS server, we will add the value manually to the Users `mycroft.conf` file.
+
+Open the file for editing.  It is not uncommon for this file to exist, but be empty.
+
+`nano ~/.config/mycroft/mycroft.conf`
+
+Enter the following into the file.  **NOTE** this file must be valid json or yaml format.  OVOS knows how to read both
```

Comment:
```suggestion
Enter the following into the file.  **NOTE** this file must be valid JSON or YAML format.  OVOS knows how to read both
```

Reply:
Consider a link to a JSON linter (https://jsonlint.com/) and/or YAML linter (https://www.yamllint.com/)